Scripts/Seleccionar.js

```diff
@@ -0,0 +1,12 @@
+/*import { redirigir, objeto } from './Modificar.js';*/
+
+function cargaIframe(Pagina) {

```

Comment:
Codacy has a fix for the issue: [Delete `␍`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456241)
```suggestion
function cargaIframe(Pagina) {
```